src/vset.h

```diff
@@ -0,0 +1,317 @@
+#ifndef VOLATILESET_H
+#define VOLATILESET_H
+
+#include <stddef.h>
+#include <stdbool.h>
+
+#include "hashtable.h"
+#include "rax.h"
+#include "sds.h"
+#include "monotonic.h" /* for mstime_t*/
+
+/*
```

Comment:
I love this documentation... but...

In the .h file, the documentation should be focused on how to USE the API.  Most of this documentation is explaining the internal structure.  I love that!  But it should be moved to the .c file.